assets/styles/style.css

```diff
@@ -0,0 +1,301 @@
+  @font-face {
+      font-family: 'Myriad Pro Regular';
+      src: url('MyriadPro-Regular.eot');
+      src: url('MyriadPro-Regular.eot?#iefix') format('embedded-opentype'), url('MyriadPro-Regular.woff') format('woff'), url('MyriadPro-Regular.ttf') format('truetype');
+      font-weight: normal;
+      font-style: normal;
+  }
+  
+  .awesome-item {
+      padding-top: 108px;
+      padding-bottom: 105px;
+  }
+  
+  .container {
+      max-width: 1100px;
+      margin-left: auto;
+      margin-right: auto;
+  }
+  
+  .title {
+      position: relative;
+      font-family: 'Raleway', sans-serif;
+      font-size: 36px;
+      text-align: center;
+      color: #212121;
+      margin: 0;
+      padding-top: 0px;
+      padding-bottom: 63px;
+  }
+  
+  .title:after {
+      position: absolute;
+      content: '';
+      display: block;
+      border-bottom: 2px solid #ff8b38;
+      width: 64px;
+      right: 518px;
+      left: 518px;
+      padding-top: 30px;
+  }
+  
+  .content {
+      text-align: center;
+      color: #86878b;
+      font-family: 'Open Sans regular', sans-serif;
+      line-height: 20px;
+      font-size: 14px;
+  }
+  
+  .awesome-block-items {
+      max-width: 1100px;
+      padding-top: 81px;
+  }
+  
+  .clearfix:after {
+      content: "";
+      display: table;
+      clear: both;
+  }
+  
+  .block-item {
+      float: left;
+      width: 31%;
+      margin-right: 3.5%;
+  }
+  
+  .block-item:last-child {
+      float: left;
+      width: 31%;
+      margin-right: 0px;
+  }
+  
+  .circle {
+      display: flex;
+      margin-left: auto;
+      margin-right: auto;
+      width: 66px;
+      height: 66px;
+      border-radius: 50%;
+      border: 2px solid#ff8b38;
+      justify-content: center;
+      align-items: center;
+  }
+  
+  .fa-lightbulb-o {
+      color: #ff8b38;
```

Comment:
the colour is the same so you can be more specific and create a separate selector